### PR TITLE
[improve] Optimize MLFlow task plugin for easy of use

### LINKED ISSUE #12070: [Improvement][Task Plugin] Optimize MLFlow task plugin for ease of use
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

- [x] Support for modifying the `dolphinscheduler-mlflow` repository
       We can modify the repository address to the local path if we can not access git
- [x] Remove deploy with `docker compose`, keep deploy with `Docker`
        `docker compose` and `DOCKER` of MLflow models are highly similar, keep the delicate one
- [x] Optimize doc
- [x] Optimize UI

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by jieguangzhou**
## Purpose of the pull request

close: #12070

- [x] Support for modifying the `dolphinscheduler-mlflow` repository
       We can modify the repository address to the local path if we can not access git
- [x] Remove deploy with `docker compose`, keep deploy with `Docker`
        `docker compose` and `DOCKER` of MLflow models are highly similar, keep the delicate one
- [x] Optimize doc
- [x] Optimize UI 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

### CODE REVIEW

File: dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 package org.apache.dolphinler.plugin.task.mlflow;
 
+import static org.powermock.api.mockito.PowerMockito.when;
```
Comment (EricGao888) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java:20]: Hello @jieguangzhou , could u plz remove `powermock` in this test class? For reason why we need to do this, see: #11405 . You could refer to  #11588 as an example.

Comment (EricGao888) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java:20]: It is totally fine to me if you think removing `powermock` in this test class is out of the scope of this PR and choose to do it in a new PR.

Comment (jieguangzhou) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java:20]: That's good advice, I wil modify some ML task plugin test cases in a new PR

**Review by zhongjiajie — APPROVED** [review]
LGTM overall

File: dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java (hunk 1 above)
Comment (zhongjiajie) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java:20]: @jieguangzhou do not forget create some issue to target remove power mock to all mlops tasks

**Review by caishunfeng — APPROVED** [review]
LGTM of the backend part

**Review by devosend — APPROVED** [review]
LGTM

**Review by Amy0104 — APPROVED** [review]
The front end part LGTM.

File: dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java (hunk 1 above)
Comment (jieguangzhou) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java:20]: I have already created a new issue about that #12077

Comment (EricGao888) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java:20]: @jieguangzhou FYI, we just added jUnit5 dependency #11332 this morning. Therefore, I will update the example I posted above with jUnit5 later this week.

Comment (jieguangzhou) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java:20]: Thank you. I'll write with reference

**Pull request merged**